openqabot/types/baseconf.py

```diff
@@ -1,17 +1,17 @@
 # Copyright SUSE LLC
 # SPDX-License-Identifier: MIT
-from abc import ABCMeta, abstractmethod, abstractstaticmethod
 from typing import Any, Dict, List, Optional
 
 from .incident import Incident
 
 
-class BaseConf(metaclass=ABCMeta):
```

Comment:
dtto

Reply:
I am closing this comment , it is just duplication of same question as above , don't see the point to copy/pasting same answer two times